backend/database/schemas.py

```
@@ -423,6 +423,19 @@ class PasswordChange(BaseModel):
     current_password: str
     new_password: str
 
+class PasswordResetRequest(BaseModel):
+    email: str
+    confirm_email: str
+    new_password: str
+
+    @model_validator(mode="after")
+    def validate_emails(self):
+        if self.email.strip().lower() != self.confirm_email.strip().lower():
+            raise ValueError("Emails must match")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)